### DISCUSSION TITLE
Thinking Forth

### DISCUSSION THREAD

**Discussion by Bruno-366** [ForthHub/discussion#101] [Show and tell]
Hey guys!
I wanted to read _Thinking Forth_ online and saw the [project website] from ~17 years ago.
Then I saw @forthy42 work at [forthy42/thinking-forth], modifying the code-examples so that they run on ANS Forth.
I wanted to [continue the work] and if anyone who speaks Spanish wants to help,
  we might be able to finish the translation to Spanish.
I'm also moving some files around to try to make the repo a bit more welcoming to newcomers.

[project website]: http://thinking-forth.sourceforge.net/
[forthy42/thinking-forth]: https://github.com/forthy42/thinking-forth
[continue the work]: https://github.com/Bruno-366/thinking-forth/tree/Bruno-366-folder

**Comment by Bruno-366** [comment]
also I was wondering if I could transfer the ownership of my fork to ForthHub

**Reply by Bruno-366** [reply]
That way the book's website would be https://forthhub.github.io/thinking-forth/

**Comment by booniepepper** [comment]
This is a fantastic book. What kind of work is needed to work on it? (I don't speak enough Spanish to help with that version)
